### PR TITLE
Fix path in testing tool

### PR DESCRIPTION
**Pull request by YuliiaKovalova**
### Problem
Fixes #https://github.com/dotnet/templating/issues/6077

### Solution
Remove usage of the virtual hive in Verification Engine

### Checks:
- [ N/A ] Added unit tests
- [ N/A ] Added `#nullable enable` to all the modified files [?](https://github.com/dotnet/templating/wiki/Contributing#coding-style)

### CODE REVIEW

File: test/Microsoft.TemplateEngine.Authoring.TemplateVerifier.IntegrationTests/VerificationEngineTests.cs [hunk 1]
```diff
@@ -31,6 +31,7 @@ public async void VerificationEngineFullDevLoop()
                 SnapshotsDirectory = snapshotsDir,
                 OutputDirectory = workingDir,
                 VerifyCommandOutput = true,
+                SettingsDirectory = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName()),
```
Comment (vlada-shubina) [inline test/Microsoft.TemplateEngine.Authoring.TemplateVerifier.IntegrationTests/VerificationEngineTests.cs:34]: did it break the test? 
why was needed?

**Review by maridematte — APPROVED** [review]
Looks good to me

**Review by vlada-shubina — COMMENTED** [review]
It will be good to add the test testing pre-installed template (if possible).

**Comment by YuliiaKovalova** [comment]
> It will be good to add the test testing pre-installed template (if possible).

i added a test. please take a look if I understood you right

**Review by vlada-shubina — APPROVED** [review]

**Pull request merged**